stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -72,6 +124,125 @@ public View getViewForHighlighting(Object element) {
     return (View)element;
   }
 
+  @Override
+  protected void onGetStyles(View element, Accumulator<StyleInfo> styles) {
+    for (Method method : viewGetters) {
+      boolean caught = false;
+      try {
+        ViewDebug.ExportedProperty annotation = getExportedAnnotation(method);
```

Comment:
let's decorate this variable with `@Nullable` since it's pretty clear that it can be `null`